packages/components/addon/components/hds/text/display.js

```diff
@@ -0,0 +1,74 @@
+import Component from '@glimmer/component';
+import { assert } from '@ember/debug';
+
+// notice: only some combinations of size + font-weight are allowed (per design specs)
+// see: https://www.figma.com/file/oQsMzMMnynfPWpMEt91OpH/HDS-Product---Foundations?node-id=1262%3A9192
+
+export const DEFAULT_SIZE = '200';
+export const AVAILABLE_SIZES = [500, 400, 300, 200, 100];
+
+// notice: the first item in the array is considered the default
+export const AVAILABLE_WEIGHTS_PER_SIZE = {
+  500: ['bold'],
+  400: ['semibold', 'medium', 'bold'],
+  300: ['semibold', 'medium', 'bold'],
```

Comment:
I would arrange these in order of visual weight:

`['semibold', 'medium', 'bold']`
=>
`['medium', 'semibold', 'bold']`

Reply:
@KristinLBradley fixed in https://github.com/hashicorp/design-system/pull/1490/commits/ce44d36ff4ffdab373d6bb1a435ad9123de7ca82

Reply:
Thanks. I do think it's clearer to define an explicit default vs. relying on order.

Reply:
Agree. Thanks for the nudging ;)